### ISSUE TITLE
Add Data Columns for CPT List Views

### ISSUE THREAD

**Issue by majick777** [netmix/radio-station#23]
Add data columns for custom post type list views
Shows - active or not, assigned DJs, assigned Genres, shift times (expandable)
Overrides - display override times
Playlists - display track count, played count and queued count

**Comment by majick777** [comment]
Minor improvement on this front, by adding `show_admin_column` to the Genre taxonomy registration, a Genre column is now automatically added to any post types that use it (ie. Show.) Also by adding `show_in_quick_edit` they can be assigned to a Show from the show list screen using the Quick Edit feature. Nifty.

**Comment by tonyzeoli** [comment]
Cool, love it!

**Comment by majick777** [comment]
So I have added some extra data columns for this one:
Shows: Active, Shift Times, DJs, Show Image
Overrides: Override Date, Start Time, End Time, Show(s) Affected, Override Image
Playlist: Track Count, Tracklist (expandable), Show

Also made the Override Date column sortable and set it to the default sort order. And added a month dropdown filter for the overrides as thought it might be useful too. Similarly, I added a weekday selector filter for the Shows list so shows with shifts on a specific weekday can be listed easily. This was a little tricky and might be all a bit of overkill, but does make it nice to have a better accessible admin interface. :-)

I think that should do it for now, unless there are other columns that would be useful to show.
